src/rantanplan/core.py

```diff
@@ -254,15 +281,20 @@ def syllabify(word):
     """
     output = ""
```

Comment:
Maybe using a list and then a join at the end would simplify a bit the code, at least it might be a bit more readable.

Reply:
Can you elaborate please?